src/tls.c

```diff
@@ -6049,7 +6086,7 @@ static int TLSX_SupportedVersions_Parse(WOLFSSL* ssl, byte* input,
             if (minor >= TLSv1_3_MINOR) {
                 if (!ssl->options.tls1_3) {
                     ssl->options.tls1_3 = 1;
-                    TLSX_Push(&ssl->extensions, TLSX_SUPPORTED_VERSIONS, ssl,
+                    TLSX_Prepend(&ssl->extensions, TLSX_SUPPORTED_VERSIONS, ssl,
```

Comment:
Shouldn't the return code here be checked and handled?

Reply:
Fixed in latest version